framework/core/src/Queue/QueueRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Queue;
+
+use Illuminate\Contracts\Queue\Queue;
+use Illuminate\Support\Str;
+
+class QueueRepository
```

Comment:
I don't think we need a new class for this especially since we have other `identifyX` private methods in the command class, or ar we using this method elsewhere?

Reply:
Oh I see, in that case, how about making this class into one that provides these system information (queue driver, scheduler status, MySQL version, session driver ..etc) so it can be used elsewhere, that way we also don't end up with this separation or multiple classes.

We could name the class something like `ApplicationInfoProvider`?

Reply:
Hmm, yeah, I like that. Will get on with that shortly